docs/manual/cluster-admin/how-to-set-up-docker-image-cache.md

```diff
@@ -0,0 +1,104 @@
+# How to Set Up Docker Image Cache
+
+[Docker Image Cache](https://docs.docker.com/registry/recipes/mirror/), implemented as docker-cache service in OpenPAI, can help admin avoid [dockerhub rate limit](https://www.docker.com/increase-rate-limits), which make deployment of service or user sumbitted job pending for a while. Docker Image Cache basically set as a pull-through cache which use [Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/) or linux filesystem as storage backend. Furthermore, with utility script to distribute docker-cache config, admins can easily switch to use their own docker registry or pull-through cache.
+
+
+## Set Up Docker Image Cache in Installation
+
+In installation, the only effort you need to perform is change config.yaml in contrib/kubespray/config.yaml. Those setting with "docker_cache" substring are related in "OpenPAI Customized Settings" section. 
+
+* `enable_docker_cache`: true if you want to enable docker-cache service, default is false, which make all following params won't take effect.
+* `docker_cache_storage_backend`: storage backend type selector, "azure" is for [Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/), "filesystem" is for linux filesystem.
+* `docker_azure_account_name`: required when storage backend is "azure", should be your azure blob storage accountname.
+* `docker_azure_account_key`: required when storage backend is "azure", should be your azure blob storage base64 encoded account key.
+* `docker_cache_azure_container_name`: required when storage backend is "azure", should be modified if you want to specify container name your docker-cache use, default is "dockerregistry".
+* `docker_cache_fs_mount_path`: required when storage backend is "filesystem", should be modified if you want to specify path your docker-cache use, default is "/var/lib/registry".
+* `docker_cache_remote_url`: pull-through cache remote url, should be modified if you want to specify other docker remote registry rather than dockerhub, default is "https://registry-1.docker.io/".
+* `docker_cache_htpasswd`: htpasswd auth info with base64 encoded, should be use with ssl when docker-cache cache some private registry as an access control method.
+
+### `config.yaml` example with azure
+
+``` yaml
+user: forexample
```

Comment:
I suggest to remove `user`, `password`, `docker_image_tag` here. Because `docker image_tag` will be updated every release.